mono/utils/win64.asm

```diff
@@ -36,6 +36,23 @@ mono_context_get_current PROC
 	mov rax, qword ptr [rsp]
 	mov [rcx + 80h], rax
 
+	movaps xmmword ptr [rcx + 90h], xmm0
```

Comment:
Replace this function with RtlCaptureContext.

Reply:
I guess that will capture more than we want for this specific scenario, so just adding whats needed into MonoContext directly without additional need to to yet another mono_sigctx_to_monoctx call. RtlCaptureContext. RtlCaptureContext is also marked as a desktop apps only API, so using existing implementation

Reply:
"More than we want":

RtlCaptureContext captures:

 rcx (it gets trashed by making the call)
 fxsave (x87, sse)
 rsp
 rip
 eflags
 cs ds es ss fs gs
 rax rdx r8 r9 r10 r11 rbx rbp rsi rdi r12 r13 r14 r15
 MxCsr
 and sets ContextFlags to CONTEXT_FULL or CONTEXT_SEGMENTS

 Possibly this is all too little in either case, yours and RtlCaptureContext, i.e. might need ymm registers.

I"ll ask to get it added for apps.